.github/workflows/build.yml

```diff
@@ -147,7 +147,9 @@ jobs:
     # find . -name "*.csproj" | xargs -I % echo dotnet pack % -c Release -o /tmp/packages -p:Version=\$GENVERSION
     - name: Pack the package VERSION
       run: |
-        dotnet pack ArmoniK.Extensions.Csharp.sln -c Release -o /tmp/packages -p:Version=$GENVERSION
+        dotnet pack ArmoniK.Extensions.Csharp.sln -c Release -p:Version=$GENVERSION
+        mkdir -p /tmp/packages
+        mv ./publish/*.* /tmp/packages/
```

Comment:
Couldn't you avoid this move and directly upload the artifacts from the publish folder?

Reply:
yes but there are 2 output possible release or normal one and I don't want to mix them

Reply:
Ah, I see. It would have worked if the publish folder was cleaned up after the first artifact upload.